github-pr.js

```diff
@@ -77,31 +78,63 @@ function getAttachLinks(bugIds, prURL, prNum, prTitle) {
 
 
 /**
- * Creates the "Attach to bug: " text and link nodes and returns them.
+ * Checks if there's already a container and if not, creates one with attach
+ * links in it.
  */
-function createAttachLinks() {
+function createAttachLinksContainer() {
+    // If this is not a pull request page, then return.
+    if (! /^https:\/\/github.com\/[^\/]+\/[^\/]+\/pull\//.test(window.location.href)) {
```

Comment:
For future reference, the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) object makes parsing and testing URLs way easier than regex magic.